### PR TITLE
chore: bump chart

### PR DESCRIPTION
**Pull request by juliangieseke**
- bumps chart
- adds trailing slash (cc @jaxesn)
- cleans up old file, we dont need to keep multiple revisions of an addon if we dont want to explicitly support them… 🤷‍♂ (cc @alejandroEsc)

### CODE REVIEW

File: addons/kommander/1.x/kommander-11.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.4.8
+    version: 0.4.9
```
Comment (makkes) [inline addons/kommander/1.x/kommander-11.yaml]:
```suggestion
    version: 0.4.10
```
I think you could save some time by bumping it to the latest version here.

Comment (shaneutt) [reply addons/kommander/1.x/kommander-11.yaml]: Let's bump this and then I'm :+1: for this change.

**Comment by jaxesn** [comment]
Do we need to update [here](https://github.com/mesosphere/kubeaddons-kommander/pull/12/files#diff-e14baf0720cc7827d0e9781de97bf5c5R75) after this https://github.com/mesosphere/charts/pull/444? cc @shaneutt @hectorj2f

**Comment by juliangieseke** [comment]
I'd say thats a follow-up PR, chart will also bumped from `.9` to `.10`

**Comment by jaxesn** [comment]
You are bumping to .10 in this PR, which includes the new catalog update?

**Comment by juliangieseke** [comment]
uh, true. not me, but @makkes @sebbrandt87 you updated the chart, can you add more details?

File: addons/kommander/1.x/kommander-11.yaml [hunk 2]
```diff
@@ -6,9 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-10"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-11"
     appversion.kubeaddons.mesosphere.io/kommander: "1.0.0"
-    endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui
+    endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui/
```
Comment (shaneutt) [inline addons/kommander/1.x/kommander-11.yaml]: :thinking: did this cause a problem somewhere?

**Comment by sebbrandt87** [comment]
> uh, true. not me, but @makkes @sebbrandt87 you updated the chart, can you add more details?

saving one extra bump, not good?!

**Comment by juliangieseke** [comment]
usually yes, but seems like there is more changes necessary with `0.4.10` (see jackons comment)?

File: addons/kommander/1.x/kommander-11.yaml (hunk 2 above)
Comment (jaxesn) [reply addons/kommander/1.x/kommander-11.yaml]: Yeah the UI doesn't handle with no trailing slash and it doesn't redirect to the include the trailing slash. We plan on fix that tho

**Comment by jaxesn** [comment]
I don't know if there is or there isn't but I suspect they may be and if that's true well just be confused later when it's not using the image tag we expected... Shane can you please check on this version of catalog and see if it needs bumped or not?

File: addons/kommander/1.x/kommander-11.yaml (hunk 2 above)
Comment (shaneutt) [reply addons/kommander/1.x/kommander-11.yaml]: :+1:

**Comment by shaneutt** [comment]
> I don't know if there is or there isn't but I suspect they may be and if that's true well just be confused later when it's not using the image tag we expected... Shane can you please check on this version of catalog and see if it needs bumped or not?

In fact I was digging into this and it does, I've got another PR we need to make it happen: https://github.com/mesosphere/charts/pull/445

**Comment by sebbrandt87** [comment]
okay I fully lost track for today I think. Sorry for raising the version, feel free to dismiss and roll it out and later do the bump to 0.4.10.
@makkes fyi ☝️

**Comment by shaneutt** [comment]
@juliangieseke https://github.com/mesosphere/charts/pull/445 is now resolved, I think we need another commander chart update to include it?

**Comment by shaneutt** [comment]
Blocked on https://github.com/mesosphere/charts/pull/447

File: addons/kommander/1.x/kommander-11.yaml [hunk 3]
```diff
@@ -41,7 +41,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.4.8
+    version: 0.4.10
```
Comment (shaneutt) [inline addons/kommander/1.x/kommander-11.yaml]: As per https://github.com/mesosphere/charts/pull/447

```suggestion
    version: 0.4.12
```

**Comment by shaneutt** [comment]
Everything upstream should be taken care of for the API update, unblocked!

**Comment by alejandroEsc** [comment]
Are we thinking the deletion of revisions properly? If this is how we are approaching this, then why even have revisions?

**Comment by shaneutt** [comment]
> Are we thinking the deletion of revisions properly? If this is how we are approaching this, then why even have revisions?

The primary reason to have revisions is to support upgrade paths. When we support downgrade paths and Kommander is GA we may need to be more stringent. https://jira.d2iq.com/browse/D2IQ-64457 is the follow up which tries to answer this question more fully.

**Comment by alejandroEsc** [comment]
> > Are we thinking the deletion of revisions properly? If this is how we are approaching this, then why even have revisions?
> 
> The primary reason to have revisions is to support upgrade paths. When we support downgrade paths and Kommander is GA we may need to be more stringent. https://jira.d2iq.com/browse/D2IQ-64457 is the follow up which tries to answer this question more fully.

Revisions were created as a request from kommander ui to allow for different versions of an addon to be accessible for installation by an end user. What you mention could be tied to versions of a release branch and would not require revisions in that case.

**Pull request merged**